index.bs

```diff
@@ -940,14 +886,13 @@ string-valued keys. Values may be any type that has a valid encoding in JSON. It
 
     This structure is used by the client to compute the following quantities:
 
-    : <dfn dfn>clientDataJSON</dfn>
-    :: This is the <a>UTF-8 encoded</a> JSON serialization [[RFC7159]] of a {{ClientData}} dictionary. Any valid JSON
-        serialization may be used by the client. This specification imposes no canonicalization requirements. Instead, the
-        {{ScopedCredentialInfo}} and {{AuthenticationAssertion}} structures contain the actual serializations used by the client
-        to generate them.
+    : <dfn dfn>JSON-serialized client data</dfn>
+    :: This is the [=UTF-8 encoding=] of the result of calling the initial value of {{JSON/stringify|JSON.stringify}} on a
+        {{ClientData}} dictionary. To avoid ambiguity, the {{ScopedCredentialInfo}} and {{AuthenticationAssertion}} structures
+        contain the actual serializations used by the client to generate them.
 
-    : <dfn dfn>clientDataHash</dfn>
-    :: This is the hash (computed using {{hashAlg}}) of <a for="ClientData">clientDataJSON</a>, as constructed by the client.
+    : <dfn dfn>Hash of the client data</dfn>
+    :: This is the hash (computed using {{hashAlg}}) of the [=JSON-serialized client data=], as constructed by the client.
```

Comment:
I also suggest we make s/client data/<dfn>client data</dfn>/ at this line: https://github.com/w3c/webauthn/pull/369/files#diff-ec9cfa5f3f35ec1f84feb2e59686c34dR858 (it wouldnt let me insert a line comment there, so doing it here) -- and then use `[=client data=]" everywhere we are referring to just (unserialized, unhashed) "client data".